paddle/fluid/platform/device_context.cc

```diff
@@ -203,6 +212,24 @@ class CudnnHolder {
   std::mutex mtx_;
 };
 
+CudnnWorkspaceHandle::CudnnWorkspaceHandle(CudnnHolder* holder)
+    : holder_(holder) {}
+
+void CudnnWorkspaceHandle::RunFunc(const std::function<void(void*)>& cudnn_func,
```

Comment:
use 
```
template <typename Callback>
void RunFunc(Callback cudnn_func) {}
```

then compiler can inline this method call.

Reply:
Done. Thanks.